packages/aws-cdk/lib/logging.ts

```diff
@@ -13,6 +13,16 @@ const logger = (stream: Writable, styles?: StyleFn[]) => (fmt: string, ...args:
   stream.write(str + '\n');
 };
 
+export const enum LogLevel {
```

Comment:
As discussed, removing `const` here is not a breaking change, and would improve chances everything works okay since `esbuild` might perform single-file transforms.

```suggestion
export enum LogLevel {
```

Reply:
Will do.